### DISCUSSION TITLE
atmos_4x_daily files overwritten on restart?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1056] [Announcements]
It looks like the atmos_4x_daily files are overwritten if you restart the model and only the data for the last run is preserved. Is this known behavior? Is there something I could set to prevent this, or does that need to be handled as part of the workflow?

**Comment by yangfanglin** [comment]
@Walter @JessicaMeixner-NOAA  Should atmos_4x_daily be removed from https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/parm_fv3diag/diag_table_cpl since it is not actually used.

@benjamin-cash,  You might be using a different diag_table which still contains atmos_4x_daily.   Wondering why you are still using it while we have atmos forecasts written out separately for each forecast hour at a given output frequency using the Write Grid Component.   We had stopped using atmos_4x_daily, written out by FMS, before GFS.v15 was implemented into operation.
